pygmt/tests/test_grdview.py

```diff
@@ -3,262 +3,258 @@
     """
-    fig = Figure()
-    fig.grdview(
-        grid=grid, cmap="oleron", surftype="s", perspective=[225, 30], zscale=0.005
-    )
-    return fig
+    fig_ref, fig_test = Figure(), Figure()
+    kwargs = dict(cmap="oleron", surftype="s", perspective=[225, 30], zscale=0.005)
+    fig_ref.grdview(grid=gridfile, **kwargs)
+    fig_test.grdview(grid=xrgrid, **kwargs)
+    return fig_ref, fig_test
 
 
-@pytest.mark.xfail(
-    reason="Baseline image not updated to use earth relief grid in GMT 6.1.0",
-)
-@pytest.mark.mpl_image_compare
-def test_grdview_on_a_plane(grid):
+@check_figures_equal()
+def test_grdview_on_a_plane(gridfile, xrgrid):
     """
     Run grdview by passing in a grid and plotting it on a z-plane, while
     setting a 3D perspective viewpoint.
     """
-    fig = Figure()
-    fig.grdview(grid=grid, plane=-4000, perspective=[225, 30], zscale=0.005)
-    return fig
+    fig_ref, fig_test = Figure(), Figure()
+    kwargs = dict(plane=-4000, perspective=[225, 30], zscale=0.005)
+    fig_ref.grdview(grid=gridfile, **kwargs)
+    fig_test.grdview(grid=xrgrid, **kwargs)
+    return fig_ref, fig_test
 
 
-@pytest.mark.xfail(
-    reason="Baseline image not updated to use earth relief grid in GMT 6.1.0",
-)
-@pytest.mark.mpl_image_compare
-def test_grdview_on_a_plane_with_colored_frontal_facade(grid):
+@check_figures_equal()
+def test_grdview_on_a_plane_with_colored_frontal_facade(gridfile, xrgrid):
     """
     Run grdview by passing in a grid and plotting it on a z-plane whose frontal
     facade is colored gray, while setting a 3D perspective viewpoint.
     """
-    fig = Figure()
-    fig.grdview(grid=grid, plane="-4000+ggray", perspective=[225, 30], zscale=0.005)
-    return fig
+    fig_ref, fig_test = Figure(), Figure()
+    kwargs = dict(plane="-4000+ggray", perspective=[225, 30], zscale=0.005)
+    fig_ref.grdview(grid=gridfile, **kwargs)
+    fig_test.grdview(grid=xrgrid, **kwargs)
+    return fig_ref, fig_test
 
 
-@pytest.mark.xfail(
-    reason="Baseline image not updated to use earth relief grid in GMT 6.1.0",
-)
-@pytest.mark.mpl_image_compare
-def test_grdview_with_perspective_and_zaxis_frame(grid):
+@check_figures_equal()
+def test_grdview_with_perspective_and_zaxis_frame(gridfile, xrgrid, region):
     """
     Run grdview by passing in a grid and plotting an annotated vertical
-    z-axis frame.
+    z-axis frame on a Transverse Mercator (T) projection.
     """
-    fig = Figure()
-    fig.grdview(grid=grid, perspective=[225, 30], zscale=0.005, frame="zaf")
-    return fig
+    fig_ref, fig_test = Figure(), Figure()
+    projection = f"T{(region[0]+region[1])/2}/{abs((region[2]+region[3])/2)}"
+    kwargs = dict(
+        projection=projection,
+        perspective=[225, 30],
+        zscale=0.005,
+        frame=["xaf", "yaf", "zaf"],
+    )
+    fig_ref.grdview(grid=gridfile, **kwargs)
+    fig_test.grdview(grid=xrgrid, **kwargs)
+    return fig_ref, fig_test
```

Comment:
Ok, added a projection (Tranverse Mercator) from the suggestions at https://github.com/GenericMappingTools/gmt/issues/4181, and this is the new reference/test plot:

![test_grdview_with_perspective_and_zaxis_frame](https://user-images.githubusercontent.com/23487320/92420130-c8ec4480-f1c5-11ea-8015-f3d6b13c446f.png)

Tests pass when using both `make test` and `pytest --verbose --mpl --doctest-modules pygmt/tests/test_grdview.py` now :tada: